src/stackFrame_aarch64.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2017, BellSoft LLC. All rights reserved.
```

Comment:
Code changes look good. Do you mind retaining non-exclusive copyrights to the project owner, too?